contracts/src/bridge/SequencerInbox.sol

```diff
@@ -61,6 +61,28 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
         _;
     }
 
+    uint256 internal immutable deployTimeChainId = block.chainid;
+
+    function _chainIdChanged() internal view returns (bool) {
+        return deployTimeChainId != block.chainid;
+    }
+
+    uint256 internal constant FORK_DELAY_BLOCKS = 1;
+    uint256 internal constant FORK_FUTURE_BLOCKS = 1;
+    uint256 internal constant FORK_DELAY_SECONDS = 1;
+    uint256 internal constant FORK_FUTURE_SECONDS = 1;
+
+    function _checkAndHandleFork() internal {
```

Comment:
this function just updates the storage variables once
we can isolate in the flow by having a permissionless external function (which allows anyone to set once a fork is identified)